README.md

```diff
@@ -34,6 +33,43 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Requirements
+
+  * Python 2.7 (the code here is not Python 3.x ready)
+  * bzip2 / buzip2
+  * Python modules: numpy, rpy2
+  * R (e.g. `sudo apt-get install r-base`)
+  * Python modules required for plotting: matplotlib, seaborn
+  * Required for generating LaTeX tables: pdflatex (e.g. `sudo apt-get install tetex-live`)
+
+### CSV format
+
+Scripts here expect to work with benchmark data in CSV format. The first row
+should contain a header with a process execution id, benchmark name and sequence
```

Comment:
I think we need to be clear that the number of in-process iterations numbers must be *exactly* the number of in-process iterations for each process execution (or bad things happen).